backend/src/models/base/abstract.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+
+from sqlalchemy import DateTime
+from sqlalchemy.ext.declarative import AbstractConcreteBase
+from sqlalchemy.orm import Mapped, mapped_column
+
+
+class TimestampAbstractModel(AbstractConcreteBase):
```

Comment:
TimestampAbstractMixin. Think about it. Again it's for your choice

Reply:
I don’t think so - it doesn’t add some logic, this is just a base model from which other models are inherited